web/customize_viewer.css

```diff
@@ -2,31 +2,160 @@
  * Custom css for our fork of pdf.js.
  */
 
+html body.light {
+  font-family: "Lato", Arial, sans-serif;
+  font-size: 13px;
+  color: #4a4a4a;
+}
+
+html body.light,
+html body.light #sidebarContainer,
+html body.light #sidebarContent,
+html body.light #mainContainer,
+html body.light #viewerContainer {
+   background: transparent;
+   box-shadow: none;
+}
+
 /**
- * Display the rotate icon in the main toolbar. We have to add this because
- * the standard viewer only showed the rotate button in the dropdown menu.
- * Transform the rotate button depending on whether we're viewing it in
- * landscape or portrait.
+ * Toolbar structure and background
  */
-.rotateCw::before {
-  content: url(images/secondaryToolbarButton-rotateCw.png);
+html body.light #toolbarContainer,
+html body.light #toolbarSidebar {
+  position: relative;
+  margin: 5px;
+  width: auto;
+  height: 50px;
+  border-radius: 4px;
+  background-image: linear-gradient(to bottom, #ffffff, #f0f0f0);
+  box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.2);
+  overflow: hidden;
 }
 
-#mainContainer.landscape .rotateCw::before {
-  /* transform: rotate(-90deg); */
+html body.light .sidebarOpen #toolbarSidebar {
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  margin-right: 0;
 }
 
-@media screen and (min-resolution: 2dppx) {
-  .rotateCw::before {
-    content: url(images/secondaryToolbarButton-rotateCw@2x.png);
-  }
+html body.light .sidebarOpen #toolbarContainer {
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+  margin-left: 0;
+}
+
+html body.light #toolbarSidebar::before {
+  content: '';
+  display: block;
+  position: absolute;
+  z-index: 10000;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: linear-gradient(to bottom, #ffffff, #f0f0f0);
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+
+html body.light #toolbarSidebar .toolbarButton {
+  position: relative;
+  z-index: 10001;
```

Comment:
The default theme sets the toolbar to `z-index:9999`, and I need these elements to display on top.

I'm trying to edit the default css files as little as possible so that it's easier to pull in upstream changes.